### PR TITLE
#899: Support slash.exclude decorator for fixtures

### PR DESCRIPTION
**Pull request by yaelmi3**

### CODE REVIEW

**Comment by vmalloc** [comment]
@yaelmi3 I think this needs a changelog entry...

File: tests/test_fixture_exclusion.py [hunk 1]
```diff
@@ -96,3 +96,26 @@ def test_invalid_exclude_patterns(names, values):
...
+    with slash.Session() as s:
+
+        @s.fixture_store.add_fixture
+        @slash.parametrize('param', [1, 2])
+        @slash.exclude("param", [1])
+        @slash.fixture
+        def fixture1(param):
+            return param
+
+        def test_something(fixture1):
```
Comment (vmalloc) [inline tests/test_fixture_exclusion.py:112]: I think this test case should have an additional fixture that is parametrized and not marked with exclude to make sure this plays well together...

Comment (yaelmi3) [reply tests/test_fixture_exclusion.py]: Fixed

**Comment by vmalloc** [comment]
Thanks!

**Pull request merged**